src/containers/SearchPage/components/form/SearchContentForm.tsx

```diff
@@ -240,7 +240,7 @@ class SearchContentForm extends Component<Props & tType, State> {
                 name={selectField.name}
                 options={selectField.options}
                 idKey="id"
-                value={this.state.search[selectField.name]}
+                value={this.state.search[selectField.name] as string} // TODO: Does this have to be cast?
```

Comment:
Kikka litt på typene og ser ut som at det krever mer arbeid enn det er vertd, må sette typen på `selectFields` og fikse litt typen til `state.search` slik at den skjønner at det er kun 4 forskjellige mulige valg `this.state.search[selectField.name]` denne kan ha og at den ikke kan dermed vært `undefined`. Du kan leke deg med typene mer hvis du vil altså